scripts/version-and-build.sh

```diff
@@ -33,12 +28,12 @@ git push --tags
 # Build docker image
 docker build -t $SERVICE_NAME:v$version . --build-arg NPM_TOKEN=${NPM_TOKEN}
 
-# Push to DockerHub
-docker tag $SERVICE_NAME:v$version dydxprotocol/$SERVICE_NAME:v$version
-docker login -u $DOCKER_HUB_USERNAME -p $DOCKER_HUB_PASSWORD
-docker push dydxprotocol/$SERVICE_NAME
-
 # Push to ECR
 $(aws ecr get-login --no-include-email --region us-east-1)
 docker tag $SERVICE_NAME:v$version $AWS_ACCOUNT_ID.dkr.ecr.us-east-1.amazonaws.com/$SERVICE_NAME:v$version
 docker push $AWS_ACCOUNT_ID.dkr.ecr.us-east-1.amazonaws.com/$SERVICE_NAME:v$version
+
```

Comment:
Not a huge deal but this will make us not have to wait on dockerhub to have our services be deployable